services/src/main/scala/cromwell/services/metadata/impl/WriteMetadataActor.scala

```diff
@@ -65,7 +65,12 @@ class WriteMetadataActor(override val batchSize: Int,
       val metadataEvents =
         metadataWriteAction.events.map { event =>
           event.value match {
-            case Some(eventVal) => event.copy(value = Option(MetadataValue(StringUtil.cleanUtf8mb4(eventVal.value))))
+            case Some(eventVal) =>
+              if (metadataKeysToClean.contains(event.key.key) && eventVal.valueType == MetadataString) {
```

Comment:
I think technically this would be faster if we evaluated `eventVal.valueType == MetadataString` first (short circuiting). Because then we can skip the multiple comparisons against `metadataKeysToClean` values.